### ISSUE TITLE
brew pull does not work with dupes repo

### ISSUE THREAD

**Issue by bw-yuhogg** [Linuxbrew/brew#105]
Using `brew pull` on a PR from the Linuxbrew/homebrew-dupes repo has three undesired effects:
1. It taps homebrew-dupes into `Library/Taps/linuxbrew/homebrew-dupes`, even though it now lives under `homebrew/homebrew-dupes`. (It only does this if it's not already tapped, although if you delete the folder it tapped, it will do it again)
2. It complains about a "Nonstandard bump subject"
3. The bottle commit download fails. (The "build a bottle for such-and-such" commit appears to be downloaded and applied correctly, albeit to the wrong location as per 1.)

Example:

``` bash
$ brew pull --bottle https://github.com/Linuxbrew/homebrew-dupes/pull/62
==> Tapping linuxbrew/dupes
Cloning into '/home/bob/.linuxbrew/Library/Taps/linuxbrew/homebrew-dupes'...
remote: Counting objects: 43, done.
remote: Compressing objects: 100% (43/43), done.
remote: Total 43 (delta 0), reused 4 (delta 0), pack-reused 0
Unpacking objects: 100% (43/43), done.
Checking connectivity... done.
Tapped 38 formulae (105 files, 341.8K)
==> Fetching patch 
Patch: https://github.com/Linuxbrew/homebrew-dupes/pull/62.patch
==> Applying patch
Applying: less: Build a bottle for Linuxbrew
==> Patch closes issue #62
Warning: Nonstandard bump subject: less: Build a bottle for Linuxbrew
Warning: Subject should be: less  (new formula)
Error: Failure while executing: /usr/bin/curl --fail --progress-bar --remote-time --location --user-agent Linuxbrew/0.9.9\ (Linux;\ x86_64\ Ubuntu\ 14.04.5\ LTS)\ curl/7.35.0 --silent --fail -o /dev/null -I https://github.com/LinuxbrewTestBot/homebrew-dupes/compare/linuxbrew:master...pr-62
```

**Comment by sjackman** [comment]
`brew pull` assumes the tap directory is the same as the GitHub repo slug, but you can tell it otherwise.

``` sh
brew pull --bottle --tap=Homebrew/dupes https://github.com/Linuxbrew/homebrew-dupes/pull/62
```

**Comment by bw-yuhogg** [comment]
That has the same effect, unfortunately. Is the tap _supposed_ to live under `Taps/homebrew` or `Taps/linuxbrew`?

I took a look at the `pull.rb` file after it failed and didn't see any options there, other than "--tap", that looked like it would help.

**Comment by sjackman** [comment]
Perhaps `brew pull --tap` is broken. I'll look into it. `Linuxbrew/dupes` is supposed to live at `Taps/homebrew/dupes` so that `depends_on "homebrew/dupes/m4" unless OS.mac?` is able to find the formula. It won't find the formula if it's tapped at `Taps/linuxbrew/dupes`.

**Comment by bw-yuhogg** [comment]
Fixed by ea6246a1d2d101ed2ad2673b50a024ab78bfbcd2. Thanks a bunch Shaun and my apologies for the screw-up.

**Comment by sjackman** [comment]
No worries at all, Bob.
